thesis-de/content-de.tex

```diff
@@ -1730,88 +1746,101 @@ \subsection{Definition eines Graphen}
-  unterstützen. Es dient dazu, dass im späteren Verlauf, wenn das Produkt
-  bereits beim Kunden eingesetzt wird, auf bestimmte Kundenwünsche direkt
-  eingegangen werden kann. Diese können dann mit schon zur Verfügung gestellten
-  Plugins zum Beispiel in der Community oder aber durch selbst programmierte
-  Plugins realisiert werden. Die Weboberfläche sollte jedoch auch andere
-  Sprachen unterstützen, um so eine Unabhängigkeit von Entwicklern
-  gewährleisten zu können. Unternehmen müssen so nicht auf spezielle Entwickler
-  von bestimmten Sprachen zurückgreifen.
-  \1 Die Datensicherung ist bei der Datenvisualisierung ein wesentlicher
-  Aspekt. Dabei muss die Weboberfläche bereits Möglichkeiten bieten, eine
-  Datensicherung des \glslink{Dashboard}{Dashboards} durchführen zu können. Das
-  Dashboard beinhaltet alle Graphen und vorkonfigurierte Werte. Diese sollen
-  über ein externes System (Dateifreigabeserver oder ähnlichem) abgelegt werden
-  und können somit einfach dupliziert und gesichert werden.
+  \1 Wie bereits in der Grundanforderung im Abschnitt
+  "Anforderungen~\ref{section:anforderungen} definiert, muss die zu
+  Evaluieriende Softwarelösung eine Open-Source-Lizenz besitzen, damit hier der
+  Quellcode angepasst werden kann und keine (Folge-)Kosten innerhalb des
+  Projekts oder später beim Kunden anfallen können.
+\end{outline}
+
+Sicherheit
+\begin{outline}
+  \1 Um die Sicherheit und den zugriff auf das Webinterface gewährleisten zu
+  können, muss an die Datenvisualisierung die Anforderung gestellt werden, das
+  sie eine Authentisierung, Authentifizierung und Autorisierung unterstützt.
+  Dies dient in erster Linie die Daten, sowie die Anmeldung an dem Webinterface
+  selbst vor unautorisiertem Zugriff durch Dritte zu schützen. Der Zugriff auf
+  das Webinterface soll mit einer Transportverschlüsselung durch \gls{HTTPS}
+  durchgeführt werden und je nach Kundenanforderung auch eigene
+  Client-Zertifikate zulassen. Die Anmeldung an der Datenvisualisierung kann
+  entweder über einen lokalen Administrator, als auch über eine LDAP-Anbindung
+  (Lightweight Directory Access Protocol) verfügen, um hier zum Beispiel
+  Unternehmensverzeichnisse der Kunden anbinden zu können. Hierrüber soll
+  ebenfalls eine Rechtevergabe innerhalb der Datenvisualisierung möglich sein.
+\end{outline}
+
+Skalierung
+\begin{outline}
+  \1 Die Datenvisualisierung muss eine Plugin-Erweiterung unterstützen. Es
+  dient dazu, dass im späteren Verlauf, wenn das Produkt bereits beim Kunden
+  eingesetzt wird, auf bestimmte Kundenwünsche direkt eingegangen werden kann.
+  Diese können dann mit schon zur Verfügung gestellten Plugins zum Beispiel in
+  der Community oder aber durch selbst programmierte Plugins realisiert werden.
+  Die Weboberfläche sollte jedoch auch andere Sprachen unterstützen, um so eine
+  Unabhängigkeit von Entwicklern gewährleisten zu können. Unternehmen müssen so
```

Comment:
ist das echt so?
Die Plugins brauchen doch immer irgendwie javascript oder css? Wir sind hier ja im webbereich?
Kann ich in Grafana echt Plugins in java, R oder C schreiben?